meta2v2/meta2_backend.c

```diff
@@ -757,6 +760,20 @@ meta2_backend_add_modified_container(struct meta2_backend_s *m2b,
 		oio_events_queue__send_overwritable(m2b->notifier,
 				sqlx_admin_get_str(sq3, SQLX_ADMIN_BASENAME),
 				_container_state (sq3));
+
+	gchar *account = sqlx_admin_get_str(sq3, SQLX_ADMIN_ACCOUNT);
+	gchar *user = sqlx_admin_get_str(sq3, SQLX_ADMIN_USERNAME);
+	struct oio_url_s *url = oio_url_empty();
+	oio_url_set(url, OIOURL_NS, m2b->ns_name);
+	oio_url_set(url, OIOURL_ACCOUNT, account);
+	oio_url_set(url, OIOURL_USER, user);
+
+	_meta2_backend_force_prepare_data(m2b,
```

Comment:
I suggest avoiding an HEXID compuation with partial information.
Please, wrap the call to  `_meta2_backend_force_prepare_data()` with a guard on `account` and `user`. Both must be non-NULL to ensure the right HEXID is computed.